Prüfschritte/de/11.5.2.16 Programmatisch steuerbare Stati und Eigenschaften.adoc

```diff
@@ -41,7 +41,8 @@ Es wird nicht geprüft, ob ein initialer Wert ausgegeben wird. Nach Interaktion
 ==== Nicht erfüllt:
 
 * Werte oder Eigenschaften von Bedienelementen sind nicht mit dem Screenreader fokussierbar oder werden bei Fokussierung nicht ausgegeben.
-* Werte sind fokussierbar und werden ausgegeben, die Änderung des Wwrtes ist jedoch nicht möglich oder der Erfolg der Änderung kann mit dem Screenreader nicht ermittelt werden.
+* Werte sind fokussierbar und werden ausgegeben, die Änderung des Wertes ist jedoch nicht möglich
```

Comment:
"Werte oder Eingenschaften... sind nicht mit dem Screenreader fokussierbar..." - Gehört das in den Prüfschritt?